definitions/scenarios/upgrade_to_satellite_6_10.rb

```diff
@@ -64,16 +64,12 @@ def check_var_lib_pulp
     end
 
     def pulp3_switchover_steps
-      add_step(Procedures::Service::Start)
```

Comment:
Instead of starting all services, then enabling pulpcore services and then starting pulpcore services only. I did,
Enable pulpcore services
Start all services